mmhuman3d/models/body_models/smplx.py

```diff
@@ -0,0 +1,247 @@
+                 keypoint_approximate: bool = False,
+                 joints_regressor: str = None,
+                 extra_joints_regressor: str = None,
+                 **kwargs):
+        """
+        Args:
+            *args: extra arguments for SMPL initialization.
+            keypoint_src: source convention of keypoints. This convention
+                is used for keypoints obtained from joint regressors.
+                Keypoints then undergo conversion into keypoint_dst
+                convention.
+            keypoint_dst: destination convention of keypoints. This convention
+                is used for keypoints in the output.
+            keypoint_approximate: whether to use approximate matching in
+                convention conversion for keypoints.
+            joints_regressor: path to joint regressor. Should be a .npy
+                file. If provided, replaces the official J_regressor of SMPL.
+            extra_joints_regressor: path to extra joint regressor. Should be
+                a .npy file. If provided, extra joints are regressed and
+                concatenated after the joints regressed with the official
+                J_regressor or joints_regressor.
+            **kwargs: extra keyword arguments for SMPL initialization.
+
+        Returns:
+            None
+        """
+        super(SMPLX, self).__init__(*args, **kwargs)
+        # joints = [JOINT_MAP[i] for i in JOINT_NAMES]
+        self.keypoint_src = keypoint_src
+        self.keypoint_dst = keypoint_dst
+        self.keypoint_approximate = keypoint_approximate
+
+        # override the default SMPL joint regressor if available
+        if joints_regressor is not None:
+            joints_regressor = torch.tensor(
+                np.load(joints_regressor), dtype=torch.float)
+            self.register_buffer('joints_regressor', joints_regressor)
+
+        # allow for extra joints to be regressed if available
+        if extra_joints_regressor is not None:
+            joints_regressor_extra = torch.tensor(
+                np.load(extra_joints_regressor), dtype=torch.float)
+            self.register_buffer('joints_regressor_extra',
+                                 joints_regressor_extra)
+
+        self.num_verts = self.get_num_verts()
+        self.num_joints = get_keypoint_num(convention=self.keypoint_dst)
+        self.body_part_segmentation = body_segmentation('smplx')
+
+    def forward(self,
+                *args,
+                return_verts: bool = True,
+                return_full_pose: bool = False,
+                **kwargs) -> dict:
+        """Forward function.
+
+        Args:
+            *args: extra arguments for SMPL
+            return_verts: whether to return vertices
+            return_full_pose: whether to return full pose parameters
+            **kwargs: extra arguments for SMPL
+
+        Returns:
+            output: contains output parameters and attributes
+        """
+
+        kwargs['get_skin'] = True
```

Comment:
What if the user expects not to get skin? 
kwargs['get_skin'] = True if 'get_skin' not in kwargs will sovle the case.

Reply:
Refer to issue #63 .